server/complex/src/config/playground.gql

```diff
@@ -0,0 +1,44 @@
+fragment TaskFields on Task { 
+    id
+    version
+    title
+    description
+    status
+}
+
+
+mutation createTask {
+  createTask(title: "Task1", description: "test", status: OPEN) {
```

Comment:
This may be not as useful. We may remove those or add one full query example for testing.